hypothesis-python/tests/cover/test_control.py

```diff
@@ -130,22 +130,20 @@ def test_current_build_context_is_current():
 
 def test_prints_all_notes_in_verbose_mode():
     # slightly roundabout because @example messes with verbosity - see #1521
-    generated_integers = []
+    messages = set()
 
-    def notefmt(x):
-        return 'x -> %d' % (x,)
-
-    @settings(verbosity=Verbosity.debug)
+    @settings(verbosity=Verbosity.debug, database=None)
     @given(integers(1, 10))
     def test(x):
-        generated_integers.append(x)
-        note(notefmt(x))
+        msg = 'x -> %d' % (x,)
+        note(msg)
+        messages.add(msg)
         assert x < 5
 
     with capture_out() as out:
         with reporting.with_reporter(reporting.default):
             with pytest.raises(AssertionError):
                 test()
     v = out.getvalue()
-    for x in generated_integers:
-        assert notefmt(x) in v
+    for x in sorted(messages):
```

Comment:
Any reason for sorting here?

Reply:
Ensures that if several entries go missing, the failure is still deterministic despite hash randomisation.